tests/disemvowel_test.js

```diff
@@ -0,0 +1,20 @@
+import chai, { expect } from 'chai'
+import { disemvowel } from '../src/disemvowel.js'
+
+describe('disemvowel()', () => {
+  it('should be a function', () => {
+    expect(disemvowel).to.be.a('function')
+  })
+  it('return string without vowels or spaces', () => {
+    const someString = 'cash me ousside'
+    expect(disemvowel(someString)).to.eql('cshmssd')
+  })
+  it('case-sensitivity is not an issue', () => {
+    const someString = 'I LOVE REG EXPRESSIONS RAWRRR'
+    expect(disemvowel(someString)).to.eql('lvrgxprssnsrwrrr')
+  })
+  it('also ignores special characters and numbers!', () => {
```

Comment:
It seems like getting rid of special characters and numbers is not disemvoweling... its like disem-everythingbutlowercase-lettering. The spec specifies "The only non a-z character you should handle is spaces." So maybe don't remove special characters and numbers?